.gitignore

```diff
@@ -39,3 +39,4 @@ publish_to_bower/
 bower_components/
 npm-debug.log
 .ember-cli
+*.swp
```

Comment:
This should generally go in a global `.gitignore`. See [here](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for info...